src/styles/Books.css

```diff
@@ -0,0 +1,191 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat&family=Roboto+Slab:wght@700&display=swap");
+
+:root {
+  --black-two: #121212;
+  --cool-blue: #4386bf;
+  --grey-border: #e8e8e8;
+}
+
+.book {
+  flex-direction: column;
+  margin: 0 6.25rem 0.938rem 6.25rem;
+  padding: 2rem 9.188rem 1.625rem 1.688rem;
+  border-radius: 4px;
+  border: solid 1px var(--grey-border);
+  background-color: var(--white);
+}
+
+.container {
+  display: flex;
+  justify-content: space-between;
+}
+
+.bookInfo {
+  max-width: 25rem;
+  display: flex;
+  flex-direction: column;
+}
+
+.title {
+  width: 100%;
+  height: 1.813rem;
+  margin: 0.188rem 20.188rem 0 0;
+  font-family: RobotoSlab, sans-serif;
+  font-size: 1.375rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.2px;
+  color: var(--black-two);
+}
+
+.author {
+  height: 1.188rem;
+  margin: 0.25rem 0 0;
+  font-family: RobotoSlab, sans-serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: var(--cool-blue);
+  cursor: pointer;
```

Comment:
- [ ] When I add `book` and `author` then It is adding the container horizontally but if you see the design then the container should add one after another like vertically, See the design and Kindly fix it 
### your design
![hiss](https://user-images.githubusercontent.com/81727988/167230567-b428af52-a50b-44b7-bd2a-67d0a17c4da0.png)
### Figma design
![figma-design](https://user-images.githubusercontent.com/81727988/167230576-e10443de-5618-4e9a-a0d0-776d0810ba40.png)